zebra-rpc/src/methods.rs

```diff
@@ -3308,6 +3311,53 @@ pub struct GetAddressBalanceResponse {
 #[deprecated(note = "Use `GetAddressBalanceResponse` instead.")]
 pub use self::GetAddressBalanceResponse as AddressBalance;
 
+/// A struct to use as parameter of `getaddressutxos`.
```

Comment:
Could we reference the method in the struct docs?

```suggestion
/// Parameters of [`RpcServer::get_address_utxos`] RPC method.
```